.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules
+build
+coverage
```

Comment:
I don't think we use this yet - might not make sense until or unless we add coverage testing